_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,354 @@
+locally, let's go ahead and deploy it.
+
+I always start with making sure that Kubectl is in the right context. In our 
+case, the context is Minikube. We can simply switch context as follows:
+
+```bash
+kubectl config use-context minikube
+```
+
+then we can deploy the container image:
+
+```bash
+kubectl run laravel-kubernetes-demo --image=yourname/laravel-kubernetes-demo 
+--port=8181 --image-pull-policy=IfNotPresent
+```
+
+The above command tells Kubectl to run our demo application from the Docker 
+image, whilst making port 8181 available for listening. The last parameter of 
+the command simply asks Kubectl to not pull the image from a registry such as
+ Docker Hub if it exists locally which in our case it does. Do note that you 
+ still need to be logged on to Docker so that Kubectl can check if the image 
+ is up to date.
+ 
+We can check that a Pod is created for our application by running:
+
+```bash
+kubectl get pods
+```
+
+which should return a similar output to:
+
+```bash
+NAME                                       READY     STATUS    RESTARTS   AGE
+laravel-kubernetes-demo-7dbb9d6b48-q54wp   1/1       Running   0          18m
+```
+
+We can also use the Minikube GUI dashboard to monitor our cluster. The GUI 
+also helps with visualising most of the discussed concepts. To view the 
+dashboard, simply run the following:
+
+```bash
+minikube dashboard
+```
+
+or to acquire the dashboard's URL address:
+
+```bash
+minikube dashboard --url=true
+```
+
+which will return the IP address and the port number of the dashboard.
+
+__Exposing a service__
+
+So far we have created a deployment which is running our application's 
+container. With the container running, we need to expose the PHP's built-in web 
+server as a service.
+
+```bash
+kubectl expose deployment laravel-kubernetes-demo --type=NodePort --port=8181
+```
+
+and provided all went well, we should see a confirmation similar to:
+
+```bash
+service "laravel-kubernetes-demo" exposed
+```
+
+We can also view the running service under "Services" navigation menu within 
+the dashboard. A more exciting way to verify this deployment and the service 
+exposure is obviously viewing our running application in the browser 😊
+
+To obtain the URL of our application (service), we can use the following 
+command:
+
+```bash
+minikube service --url=true laravel-kubernetes-demo
+```
+
+which will output the IP address and port number similar to:
+
+```bash
+http://192.168.99.101:31399
+```
+
+or, launch the application directly in the browser:
+
+```bash
+minikube service laravel-kubernetes-demo
+```
+
+__Scaling__
+
+And that is it. We have successfully deployed our application in Kubernetes. 
+It's exciting, but what's the point of doing all of this? Well, we only have 
+one deployment with a single Pod running, provisioned to a Node with the 
+exposed web service. Let's replicate this service to two more Pods acting as 
```

Comment:
You're replicating the pods. I think the word service here is confusing. Is the service _the application_ or _the kubernetes service_?